docs/root/configuration/well_known_dynamic_metadata.rst

```diff
@@ -0,0 +1,20 @@
+.. _well_known_dynamic_metadata:
+
+Well Known Dynamic Metadata
+===========================
+
+Filters can emit dynamic metadata via the `StreamInfo` interface on a `Connection`. Following are
```

Comment:
Is there anything more we can link to here? Can we describe how this might be used? For example RBAC, logging, etc.? With links to make it more clear?

Reply:
Addressed in eaf0e89.